### PR TITLE
Home Page and About Page

### PR DESCRIPTION
**Pull request by basnetlaxmi**
### In this milestone, I have :

- Created Global Hackathon Website
- Added home page and give style to it
- Added about page and give style to it
- Created 'Featured Speaker ' section dynamically with js
- Used bootstrap and media-query to make responsive
- Added hamburger menu

Here is my [video link](https://www.loom.com/share/284327174ecf47e89b3a73341dce1d9f) about the description of the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                    <nav class="navbar navbar-expand-md navbar-md-light bg-light">
+                        <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                            <a href="#">
+                                <img src="./images/close.png" id="close-ham-img" alt="closeX">
+                            </a>
+                            <a href="#" class="navbar-brand">Laxmi</a>
+                            
+                            <ul class="navbar-nav ms-auto">
+                                <li class="nav-item">
+                                    <a class="nav-link active" aria-current="page" href="./pages/about.html">About</a>
```
Comment (Lameck1) [inline index.html]: - [ ] This link leads to no page in your project. This happens when the home page is open in mobile view and the user clicks on the About link in the mobile menu. Refactor it to:
```suggestion
<a class="nav-link active" aria-current="page" href="about.html">About</a>
```

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse learning partners
+- Microverse
```
Comment (Lameck1) [inline README.md]: - [ ] Credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), is missing in the Acknowledgements. Please add it as it is a requirement by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 3]
```diff
@@ -1 +1,60 @@
...
+
+
+## Built With
+
+- HTML, CSS
+- Linters, Markdown
+- GitHub Flow 
+
+## Live Demo
+ [Link to My Portfolio](https://basnetlaxmi.github.io/my-portfolio/)
```
Comment (Lameck1) [inline README.md]: - [ ] The Live Demo link should be a link to the deployed version of this project, not the developer portfolio. Please deploy this project to GitHub pages and provide the live version link in this section of the README.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+  font-size: 20px;
+}
+
+.up-footer {
+  height: 250px;
+}
+
+.down-footer {
+  background: #d3d3d3;
+}
```
Comment (Lameck1) [inline css/style.css]: - [ ] The footer's background should be white on the Mobile version of the page and dark on the desktop version of the page. Look at the screenshot below to help you notice the differences and make necessary adjustments. Take care of the font colors too.

| The Provide Design |
| --- |
| footer-desktop-design |

| Your implementation |
| --- |
| footer-imp |

File: css/about.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  .events .events-images .second-event {
+    height: 30%;
+    width: 90%;
+  }
+
+  .up-footer {
+    display: flex;
+  }
```
Comment (Lameck1) [inline css/about.css:89]: - How about we do a little improvement in this section. The implementation and the design slightly don't match.

| The Provide Design |
| --- |
| partner-design |

| Your implementation |
| --- |
| partner-imp |

## What to improve in this section
- The background-color
- Cente- align the Partner logos and reduce the margins between them.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+  margin: 10px 0;
+  width: auto;
+}
+
+#close-ham-img {
+  width: 25px;
+  margin: 45px 0;
+}
+
+.speaker-section .speakers .speaker-info {
```
Comment (Lameck1) [inline css/style.css:212]: - [ ] This speaker section looks very good. We only need to add margins/paddings to the left and to the right or reduce the width of the `div` with the class of `.speakers` and give a margin value of `margin: 0 auto` to center it within the page.

The problem here is that the implementation stretches the whole width of the page. Try and work on this.

| The Provide Design |
| --- |
| featured-design |

| Your implementation |
| --- |
| featured-imp |

File: about.html [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+      </section>
+    
+      <section class="events d-flex flex-column align-items-center m-5">
+        <h4 class="font">See the past Global Hackathons</h4>
+        <p>Take a look at the last two SD Global Summits which took place in Buenos Aires and in Warsaw.</p>
+        <div class="events-images d-flex m-5">
+          <img src="./images/event2.png" alt="first-event" class="first-event">
+          <img src="./images/event1.png" alt="second-event" class="second-event">
+        </div>
+      </section>
```
Comment (Lameck1) [inline about.html]: - [ ] This section's design for the mobile version of the about page is not in line with the provide design. Look at the differences in the screenshot below and make the necessary adjustments.

| The Provide design | Your Implementation |
| --- | --- |
| past-gloabal-design | past-global-imp |

- Text alignment for sections of the about page should be center in the mobile version.
- The two banners are not properly sized and positioned as in the provided design.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @basnetlaxmi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Add the node-modules directory to .gitignore file.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @basnetlaxmi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**